src/components/home/home.jsx

```diff
@@ -80,6 +80,14 @@ class HomePage extends React.Component {
               </ul>
         </section>
 
+        <section id="home-volunteer">
+            <h1>Want to join the team?</h1>
+            <p>We welcome volunteers!</p>
+            <p>We work remotely and asynchronously using online collaboration tools like GitHub.</p>
+            <p>Our stack is JavaScript/NodeJS, React, React-Native, and MongoDB.</p>
+            <button onClick={this.runMailPopup} className="cta-btn">Volunteer</button>
```

Comment:
Let's link this to the volunteer form: https://forms.gle/4ERxvyu5ThMC7dGU7

Reply:
Also, invert the look of the button so it's different from the subscribe button. Either red border with white background or teal border with white background would make it more secondary.